google/cloud/spanner/BUILD.bazel

```diff
@@ -77,6 +77,18 @@ cc_library(
     ],
 )
 
+# TODO(#3701): Delete this target after 2023-04-01.
```

Comment:
Hopefully what we'll also do is rename `spanner_client_testing_private` back to `spanner_client_testing`.

Reply:
Agreed, but I didn't want to capture all those details in the TODO.